frontend/src/components/MetDataSection/index.tsx

```diff
@@ -3,50 +3,50 @@
 import Link from "next/link";
 
 export default function MetDataSection() {
-return (
-    <main className="mb-10">
-      <div className="justify-center items-center m-4 mt-10 lg:mt-20">
-        <div className=" text-center sm:px-16 xl:px-48">
-          <h1 className="mb-4  text-3xl font-extrabold leading-none tracking-normal text-gray-700 md:text-4xl lg:text-4xl ">
+  return (
+    <main className="lg:my-10 bg-[#067A91]">
+      <div className="flex flex-col justify-center items-center text-white m-4 lg:mt-20">
+        <section className="text-center sm:px-16 xl:px-48">
+          <h1 className="mb-4 text-3xl font-extrabold leading-none tracking-normal text-white md:text-4xl lg:text-4xl">
             Met Data
           </h1>
-          <p className="mb-6 text-left font-normal text-gray-600 lg:text-xl   leading-7">
-          The India Water Portal serves as a treasure of information, housing meteorological datasets for all districts of India
-          spanning over the 20th Century `(1901-2002)`. This rich resource, encompassing 13 crucial climate parameters, offers 
-          invaluable insights for researchers, water resource managers, and anyone interested in understanding India &apos; s 
-          water story. This data can be useful in making rainwater harvesting and water balance estimates, in various research 
-          areas, climate change adaptation studies and more.
+          <p className="mb-6 text-center font-normal text-white lg:text-xl leading-7">
```

Comment:
<img width="1050" alt="Screenshot 2024-08-31 at 10 07 34 PM" src="https://github.com/user-attachments/assets/2b6b0eb2-075f-46ff-98dc-e2420a07481a">

@Kushdapush We can attempt to justify the text for improved readability and slightly reduce the font-weight of this description to enhance visual balance. Try 200 maybe.

Reply:
@sbhaya i believe that making the text even lighter will make it more difficult to read, in the below Screenshot we can see that a white font weight 400 makes the text clearer as compared to white 300 against a blueish background

![image](https://github.com/user-attachments/assets/14839c1d-c03b-4a2c-8c49-b4bdac5850b3)

Reply:
@Kushdapush Agreeing with that remark of yours. The integration of **Noto Serif** font style and **justified text** will enhance the aesthetics of this text.